app/models/submission.js

```diff
@@ -85,5 +85,20 @@ export default DS.Model.extend({
   isDraft: computed('submitted', 'submissionStatus', function () {
     return this.get('submissionStatus') === 'draft';
     // return !this.get('submitted') && !this.get('submissionStatus');
+  }),
+
+  /**
+   * @returns {boolean} is this a covid related submission?
+   */
+  isCovid: computed('metadata', function () {
+    let metadata = get(this, 'metadata') ? JSON.parse(get(this, 'metadata')) : {};
```

Comment:
Just a comment:
Would be nice to not have to worry about doing `JSON.parse` when dealing with metadata